Create new branch for project review

rmiyoyo opened the pull request:
# Work Done For Project Review
- Created the project-review-branch for review

rmiyoyo committed:
Create new branch for project review

rmiyoyo commented on index.html:
```diff
@@ -0,0 +1,95 @@
+      </div>
+      <div>
+
+      </div><blockquote> "Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress.
```
Please remove this extra ```</div>``` tag as it does not have a corresponding opening tag.

Lizdev-05 replied:
Great job 👍

rmiyoyo commented on index.html:
```diff
@@ -0,0 +1,95 @@
+      <h3 id="sub-title">The man who saved a billion lives</h3>
+      
+      <div id="img-div">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="" id="image">
```
Please consider having a descriptive value for the ```alt``` attribute of the ```a``` tag to improve accessibility for users with disabilities. :+1:

Lizdev-05 replied:
Kindly make sure that the comment has a suggestion/polite tone so that the student will not feel commanded. 😉 😊 
something like: I think it will be better to use...
or, kindly make use of...

rmiyoyo replied:
- Comment edited to reflect changes

rmiyoyo commented on index.html:
```diff
@@ -0,0 +1,95 @@
+      </div>
+
+      <h3>
+        If you have time, you should read more about this incredible human being on his <a href="https://en.wikipedia.org/wiki/Norman_Borlaug" alt ="Wikipedia entry" id="tribute-link" target="_blank">Wikipedia entry</a>.
```
The ```target="_blank"``` attribute of the ```a``` tag could potentially be a security risk as it can allow malicious sites to abuse the new window. I would suggest you use ```rel="noopener noreferrer"``` to prevent this risk.

Lizdev-05 replied:
Great one 👍

rmiyoyo commented on css/style.css:
# General Comments On CSS File
- It's important to keep the code well-indented to make it more readable and easier to understand. Please make sure each nested block is indented with a consistent number of spaces or tabs :smile: 
- The ID selectors are used throughout the code, which could cause specificity issues and make it harder to maintain the code. Kindly using classes or other types of selectors instead. :+1: 
- Naming conventions: Consider using more descriptive class names for the elements in the HTML, which will make the CSS more maintainable and easier to understand. :+1:

rmiyoyo commented on css/style.css:
```diff
@@ -0,0 +1,49 @@
+    font-size: 16px;
+}
+
+/* Tablet styles */
+@media (min-width: 768px) {
+    body {
+        font-size: 18px;
+    }
+}
+
+/* Desktop styles */
+@media (min-width: 1024px) {
+    body {
+        font-size: 20px;
+    }
+}
```
- The media queries used are targeting specific screen sizes, but there are a lot of devices with different screen sizes. Consider using breakpoints based on content and design needs rather than specific device sizes.

rmiyoyo commented on index.html:
# General Comments on HTML File
- Consider using more descriptive class names for the elements in the HTML, which will make the CSS more maintainable and easier to understand.
- The HTML structure should be properly indented and have consistent spacing to improve readability.
- I would suggest you avoid inline styles, and instead, use a separate CSS file to style the page. This makes it easier to maintain and update the styles in the future. :+1:

rmiyoyo reviewed (commented):
# Changes Required :1234: 
- Please consider the changes I have suggested.
- Also kindly install linters for HTML, CSS and JavaScript.
- I would also suggest you use the Microverse README template as it gives a better overview of your project :+1:

Lizdev-05 reviewed (changes requested):
Hi @rmiyoyo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media2.giphy.com/media/YYht2UTV41u1vxHTss/giphy-downsized-medium.gif"/>
## Highlights
- Reasons for the request are made known 👍 
- Inline comments used 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- I think you can also make use of emojis 😄 and gifs to make the code review more human-friendly :alien::robot:.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rmiyoyo commented:
# Work Done After First Review
- Changed my tone in the review. :heavy_check_mark: 
- Used emojis to make the review more friendly. :heavy_check_mark: 
- Added a comment on the need to use the Microverse README template. :heavy_check_mark:

RaoAkif reviewed (approved):
## STATUS: APPROVED 🎉🎉🎉

Hi @rmiyoyo,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![approve-image](https://ci5.googleusercontent.com/proxy/zq80mRZVpq-eAhdzmvNYP-LJmESV4U2mCLkUDj-J77H6FKB6oLO-eL8kGFENlnk15aQQPnJ2K7hSGZiaZBHMKFZEM6yCAT5H0leJxszpiJ_ZRxNc5L9uv6Wf0BFZUomZYiYN6ppiHgltD6qjc_Kh8RbH3b4nVLPghJlpfxM29IAi_q8djloxFmcQQ_I2iIY_rYXVUeBiSfNsVRfk0iq3wo75_nf9QHejNs5EwcDwp7n4vukAV0rFF6xBnAenIBlJ9_CuSGN37pZ0FDROc_LU_rInohwgD5sTJnITUxeXsq5juhu5btEu0AhtqpFV5MDVTQMfZw=s0-d-e1-ft#https://camo.githubusercontent.com/b4e8f035e0f2d20302034402026b8df675e3a9dd93b722d1b0a047112e68044d/68747470733a2f2f632e74656e6f722e636f6d2f4a52665739444c534f764141414141432f7365616c2d7365616c2d6f662d617070726f76616c2e676966)

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
